t/specialtest/runtest.py

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import random
+import subprocess
+import shutil
+import time
+import datetime
+import sys
+import os
+import json
+import re
+import base64
+from sys import version_info
+
+
+def cur_file_dir():
+    return os.path.split(os.path.realpath(__file__))[0]
+
+
+def env_prepare(casedirpath):
+    updatepip = "curl https://bootstrap.pypa.io/get-pip.py | python"
+    if version_info.major == 2:
```

Comment:
Why should it be compatible with python2.0?

Reply:
The python version of the CI environment is 2.0, but I think it is better to be compatible with 3.0.